asset/src/_kafka_clients/consumer-client.ts

```diff
@@ -95,6 +135,19 @@ export default class ConsumerClient extends BaseClient<kafka.KafkaConsumer> {
         }
     }
 
+    async onRetry(): Promise<void> {
```

Comment:
I think calling this just `retry` would make a lot more sense from the files perspective.